R/SCE2AnnData.R

```diff
@@ -15,13 +15,16 @@
 #' @export
 #' @importFrom utils capture.output
 #' @importFrom S4Vectors metadata
-#' @importFrom reticulate import r_to_py
+#' @importFrom reticulate import r_to_py py_to_r
 SCE2AnnData <- function(sce, X_name = NULL, assays = TRUE, colData = TRUE,
                         rowData = TRUE, varm = TRUE, reducedDims = TRUE,
                         metadata = TRUE, colPairs = TRUE, rowPairs = TRUE,
                         skip_assays = FALSE, verbose = NULL) {
     anndata <- import("anndata")
 
+    # create output data structure
+    adata <- list()
+
```

Comment:
I think this makes sense, any tricks to this I should know about?

Reply:
I wonder if it would be clear to name it something like `adata_list` though just so it's super obvious that this isn't an AnnData object?